semantic-convention-utils/src/main/java/org/hypertrace/semantic/convention/utils/db/DbSemanticConventionUtils.java

```diff
@@ -101,6 +102,49 @@ public static boolean isMongoBackend(Event event) {
     return Lists.newArrayList(Sets.newHashSet(OTEL_DB_OPERATION));
   }
 
+  public static List<String> getAttributeKeysForDbStatement() {
+    return Lists.newArrayList(Sets.newHashSet(OTEL_DB_STATEMENT));
+  }
+
+  public static String getOtelDbOperation(Event event) {
+    String dbOperation =
```

Comment:
Return first if the value is available, not need to do string parsing.
So, would suggest something like as below
```
public static String getOtelDbOperation(Event event) {
    String dbOperation =
        SpanAttributeUtils.getFirstAvailableStringAttribute(
            event, DbSemanticConventionUtils.getAttributeKeysForDbOperation());
    if (dbOperation != null) {
      return dbOperation;
    }

    String dbStatement =
        SpanAttributeUtils.getFirstAvailableStringAttribute(
            event, DbSemanticConventionUtils.getAttributeKeysForDbStatement());

    return getOperationFromSQLStatement(dbStatement);
  }

  private static String getOperationFromSQLStatement(String sql) {
    return StringUtils.substringBefore(StringUtils.trim(sql), " ");
  }
  
```

Reply:
And also substringbefore might return the same string if there is no space found. So, would suggest using split here.